front/app/containers/LandingPage/SignedOutHeader/HeaderContent.tsx

```diff
@@ -0,0 +1,205 @@
+  displayHeaderAvatars: boolean;
+}>`
+  width: 100%;
+  color: ${({ hasHeader, fontColors, theme }) =>
+    hasHeader
+      ? fontColors === 'light'
+        ? '#fff'
+        : theme.colorMain
+      : theme.colorMain};
+  font-size: ${fontSizes.large}px;
+  line-height: 28px;
+  font-weight: 400;
+  text-align: ${({ align }) => align};
+  text-decoration: none;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  word-break: break-word;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 15px;
+
+  ${({ displayHeaderAvatars }) =>
+    // needed because we don't always
+    // show avatars
+    !displayHeaderAvatars &&
+    `
+      margin-bottom: 38px;
+
+      ${media.smallerThanMinTablet`
+        margin-bottom: 30px;
+      `}
+  `}
+`;
+
+const StyledAvatarBubbles = styled(AvatarBubbles)`
+  min-height: 40px;
+  margin-bottom: 30px;
+
+  ${media.smallerThanMinTablet`
+    margin-bottom: 30px;
+  `}
+`;
+
+const SignUpButton = styled(Button)``;
+
+type TAlign = 'center' | 'left';
+interface Props {
+  fontColors: 'light' | 'dark';
+  align?: TAlign;
+}
+
+function getButtonStyle(fontColors: 'light' | 'dark') {
+  if (fontColors === 'light') {
+    return 'primary-inverse';
+  }
+  if (fontColors === 'dark') {
+    return 'primary';
+  }
+
+  return undefined;
+}
+
+function getAlignItems(align: TAlign) {
+  if (align === 'center') return 'center';
+  if (align === 'left') return 'flex-start';
+
+  return undefined;
+}
+
+const HeaderContent = ({
+  align = 'center',
+  fontColors,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  const appConfiguration = useAppConfiguration();
+  const localize = useLocalize();
+
+  const signUpIn = (event: React.FormEvent) => {
+    event.preventDefault();
+    trackEventByName(tracks.clickCreateAccountCTA, {
+      extra: { location: 'signed-out header' },
+    });
+    openSignUpInModal();
+  };
+
+  if (!isNilOrError(appConfiguration)) {
+    const coreSettings = appConfiguration.data.attributes.settings.core;
+    const headerTitle = coreSettings.header_title
+      ? localize(coreSettings.header_title)
+      : formatMessage(messages.titleCity);
+    const headerSubtitle = coreSettings.header_slogan
+      ? localize(coreSettings.header_slogan)
+      : formatMessage(messages.subtitleCity);
+    const headerImage = appConfiguration.data.attributes.header_bg?.large;
+    const displayHeaderAvatars =
+      appConfiguration.data.attributes.settings.core.display_header_avatars;
+    const buttonStyle = getButtonStyle(fontColors);
+
+    return (
+      <Container
+        id="hook-header-content"
+        className="e2e-signed-out-header-title"
+        alignTo={getAlignItems(align)}
+      >
+        <HeaderTitle
+          hasHeader={!!headerImage}
+          fontColors={fontColors}
+          align={align}
+        >
+          {headerTitle}
+        </HeaderTitle>
+
+        <HeaderSubtitle
+          hasHeader={!!headerImage}
+          className="e2e-signed-out-header-subtitle"
```

Comment:
Why do you add `e2e-...` class? Wasn't it used in other components because those components need to be queried by e2e tests? Do you need this class for e2e tests?

Reply:
Just copy pasted. You can find where we use it by searching for `e2e-signed-out-header-subtitle`.

e2e isn't a required name, just an easy way to recognize elements used for e2e tests. Would you like to add this to the front-end docs, end of the document?

Reply:
> Just copy pasted

Ah, I see. Thanks.

> Would you like to add this to the front-end docs, end of the document?

I think it's pretty self-explanatory. I just didn't notice you copied it because that deleted file was hidden here on Github.